egs2/swbd/asr1/RESULTS.md

```diff
@@ -0,0 +1,25 @@
+<!-- Generated by scripts/utils/show_asr_result.sh -->
+# RESULTS
+## Environments
+- date: `Fri May 14 07:43:22 UTC 2021`
+- model link: https://drive.google.com/drive/folders/18jJyFxSS9nhS51H8xWkMiDybGKrKPVEV?usp=sharing
```

Comment:
Can you provide a model through zenodo?
Please check https://github.com/espnet/espnet_model_zoo#espnet-model-zoo

Reply:
Sorry, I just saw it. I would upload it through model-zoo.